package/network/services/hostapd/files/hostapd.sh

```diff
@@ -766,6 +766,9 @@ wpa_supplicant_set_fixed_freq() {
 		HT20|VHT20) append network_data "disable_ht40=1" "$N$T";;
 		HT40*|VHT40*|VHT80*|VHT160*) append network_data "ht40=1" "$N$T";;
 	esac
+	case "$htmode" in
+		NOHT*|HT*) append network_data "disable_vht=1" "$N$T";;
```

Comment:
You could actually merge this case with the one following it.

Reply:
Yes, you can use the default of the statement below on line 776:
```
*) append network_data "max_oper_chwidth=0" "$N$T";;
```
and turn that into
```
VHT20|VHT40) append network_data "max_oper_chwidth=0" "$N$T";;
*) append network_data "disable_vht=1" "$N$T";;
```